app/decorators/hash_diff_decorator/base.rb

```diff
@@ -19,13 +19,21 @@ def to_html
     def diff_val(val)
       if val.nil?
         "[nil]"
+      elsif val.is_a?(Numeric)
+        diff_numeric(val)
       elsif val.empty?
         "[empty]"
-      elsif val.is_a?(Numeric)
-        format('%.2f', val)
       else
         val.inspect
       end
     end
+
+    def diff_numeric(val)
+      if val.is_a?(Fixnum)
+        val.to_s
+      elsif val.is_a?(Numeric)
+        format("%.2f", val)
```

Comment:
is this is always a float, perhaps just call `.round(2)` ? Not sure if that would be better, just throwing it out there